Build fails due to missing `zod` dependency

### Description
A fresh install followed by build fails because `zod` is imported but not listed in dependencies.

### Steps to Reproduce
1. Clone the repository
2. Run `pnpm install`
3. Run `pnpm build`

### Actual Behavior
TypeScript compilation fails with:
Cannot find module 'zod' or its corresponding type declarations


Affected files:
- tools/bridge.ts
- tools/contract.ts
- tools/stake.ts
- tools/stellar.ts

### Expected Behavior
Build should succeed after installing dependencies.

### Suggested Fix
Add `zod` to `dependencies` in `package.json`.

--- a/package.json
+++ b/package.json
@@ -26,7 +26,8 @@
     "big.js": "^7.0.1",
     "dotenv": "^16.4.7",
     "stellar-sdk": "^13.3.0",
-    "typescript": "^5.8.3"
+    "typescript": "^5.8.3",
+    "zod": "^4.3.6"
   },
   "devDependencies": {
     "@types/big.js": "^6.2.2",
@@ -39,5 +40,6 @@
   "bugs": {
     "url": "https://github.com/Rabdi-X-Ghewar/stellarTools/issues"
   },
-  "homepage": "https://github.com/Rabdi-X-Ghewar/stellarTools#readme"
+  "homepage": "https://github.com/Rabdi-X-Ghewar/stellarTools#readme",
+  "packageManager": "pnpm@10.28.2+sha512.41872f037ad22f7348e3b1debbaf7e867cfd448f2726d9cf74c08f19507c31d2c8e7a11525b983febc2df640b5438dee6023ebb1f84ed43cc2d654d2bc326264"
 }

